src/ServiceManager/Travely.ServiceManager.Service/Protos/activity.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+import "google/protobuf/wrappers.proto";
+
+option csharp_namespace = "Travely.ServiceManager.Service";
+
+package activity;
+
+service ActivityProto {
+	rpc CreateActivity (Activity) returns (ActivityResponce);
+	rpc GetActivities(GetActivitiesRequest) returns (Activities);
+	rpc DeleteActivity (DeleteActivityRequest) returns (ActivityResponce);
+	rpc EditActivity (Activity) returns (ActivityResponce);
+}
+
+message ActivityType {
+	google.protobuf.Int64Value id = 1;
```

Comment:
isn't this equivalent to `long?` ? (Nullable)
If so, are you sure you meant using it?

Reply:
yes, for first time when entering new type - only name will be sent